pom.xml

```diff
@@ -327,20 +305,49 @@
 	</build>
 
 	<profiles>
+		<profile>
+			<id>default</id>
+			<activation>
+				<property>
+					<name>!release</name>
+				</property>
+			</activation>
+			<modules>
+				<!-- Support Modules-->
+				<module>spring-cloud-gcp-starters</module>
+				<module>spring-cloud-gcp-dependencies</module>
+
+				<!-- Production Libraries -->
+				<module>spring-cloud-gcp-autoconfigure</module>
+				<module>spring-cloud-gcp-bigquery</module>
+				<module>spring-cloud-gcp-cloudfoundry</module>
+				<module>spring-cloud-gcp-core</module>
+				<module>spring-cloud-gcp-data-datastore</module>
+				<module>spring-cloud-gcp-data-firestore</module>
+				<module>spring-cloud-gcp-data-spanner</module>
+				<module>spring-cloud-gcp-logging</module>
+				<module>spring-cloud-gcp-pubsub</module>
+				<module>spring-cloud-gcp-pubsub-stream-binder</module>
+				<module>spring-cloud-gcp-security-iap</module>
+				<module>spring-cloud-gcp-storage</module>
+				<module>spring-cloud-gcp-secretmanager</module>
+				<module>spring-cloud-gcp-security-firebase</module>
+				<module>spring-cloud-gcp-vision</module>
+
+				<!-- Docs and Samples -->
+				<module>docs</module>
+				<module>spring-cloud-gcp-samples</module>
+			</modules>
+		</profile>
+
 		<profile>
 			<id>spring</id>
+			<activation>
+				<property>
+					<name>!excludeSpring</name>
```

Comment:
This profile used to be disabled by default. (Presence in Maven wrapper config doesn't count, as that doesn't get released.) Is that possible?

Reply:
In looking at this I ended up backing out this change since I only went down that road to try to get `activateByDefault` working.